src/task-8.js

```diff
@@ -1,6 +1,40 @@
 
 export default function getTopLetter(str) {
-    // Change me!
-
-    return "";
+/*    const dop = [];
+    let key;
+    let max = 0;
+    for (let i = 0; i < str.length; i++) {
+        if (!dop[ str[i]]) {
+            dop[ str[i]] = 1;
+        } else {
+            dop[ str[i]] += 1;
+        }
+        if (dop[ str[i]] > max) {
+            max = dop[ str[i]];
+            key = str[i];
+        }
+    }
+    return key;
+}
+*/
+    const [...z] = str.split("");
+    const map = new Map();
+    for (const i in z) {
```

Comment:
Can we use the same programming paradigm? when it's possible. In this case, we have a for loop, here and below we have forEach. Please stick to one paradigm. I suggest using forEach to iterate over an array.

And why do we use `for in` for integrable objects?